coding-the-legislation/10_basic_example.md

```diff
@@ -17,19 +17,30 @@ class flat_tax_on_salary(Variable):
 ```
 
 Let's explain in details the different parts of the code:
-- `class flat_tax_on_salary(Variable):` declares a new variable with the name `flat_tax_on_salary`.
-- Metadatas:
-  - `value_type = float` declares the type of the variable. Possible types are the basic python types:
+
+### The name of the variable
+
+`class flat_tax_on_salary(Variable):` declares a new variable with the name `flat_tax_on_salary` in accordance with our [naming conventions](../contribute/variables-naming.md).
```

Comment:
Not sure about mentioning our naming good practices (that are right-now really France-centric) as authoritatively in a really basic introduction tutorial. Reading that sentence, I understand that the naming conventions are technical constraints, which they are not at all.

Maybe something like that could be clearer:

`class flat_tax_on_salary(Variable):` declares a new variable named `flat_tax_on_salary`.  You can check out our recommended [naming conventions](../contribute/variables-naming.md).